src/nIsForName.js

```diff
@@ -21,8 +21,14 @@
  *
  * @return {string[]}
  */
-function nIsForName(names) {
-  // write code here
+function nIsForName(arrayOfName = []) {
+  const hasNFirst = arrayOfName.reduce((hasNFirst, person) => {
+    if (person.split(' ').find(word => word[0] === 'N')) {
+      hasNFirst.push(person);
```

Comment:
if you want to use reduce - do not push, do it on the immutable way and return new array `return [...hasNFirst, person];`